BlogTemplate.Tests/Pages/NewTests.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using BlogTemplate.Models;
+using Xunit;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using BlogTemplate.Tests.Fakes;
+
+namespace BlogTemplate.Tests.Pages
+{
+    class NewTests
+    {
+        [Fact]
+        public void SavePost_NoExcerptIsEntered_AutoGenerateExcerpt()
+        {
+            IFileSystem fakeFileSystem = new FakeFileSystem();
+            BlogDataStore testDataStore = new BlogDataStore(fakeFileSystem);
+
+            Post newPost = new Post
+            {
+                Title = "Title",
+                Body = "This is the body of my post",
+            };
+
+            testDataStore.SavePost(newPost);
+
+            Assert.True(fakeFileSystem.FileExists($"BlogFiles\\Title.xml"));

```

Comment:
There is no need to check whether the file exists, I think that is a separate test which we already have